tools/common_lib/src/gemm.h

```diff
@@ -2461,7 +2480,7 @@ class GemmCmDispatcher : public GemmBaseDispatcher
 
     void initialize(ID3D12GraphicsCommandList* cmd_list, D3D12_CPU_DESCRIPTOR_HANDLE cpu_handle, D3D12_GPU_DESCRIPTOR_HANDLE gpu_handle)
     {
-
+        assert(!use_stateless_);
```

Comment:
Is this assert needed here?

Reply:
removed, it has already be checked in first options.